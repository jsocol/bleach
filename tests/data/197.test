<div id="130"><math href="javascript:alert(130)">CLICKME</math>
<math>
<!-- up to FF 13 -->
<maction actiontype="statusline#http://google.com" xlink:href="javascript:alert(2)">CLICKME</maction>

<!-- FF 14+ -->
<maction actiontype="statusline" xlink:href="javascript:alert(3)">CLICKME<mtext>http://http://google.com</mtext></maction>
</math>//["'`-->]]>]</div>
