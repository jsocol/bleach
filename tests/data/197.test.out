<div id="130"><math>CLICKME</math>
<math>





</math>//["'`--&gt;]]&gt;]</div>
